README.md

```diff
@@ -74,7 +74,7 @@
 ## 🚀 Live Demo and video <a name="live-demo"></a>
 https://air-quality-lyqt.onrender.com/
 
-- [Click Here](https://www.loom.com/share/976cd9a30fd54fe799e36a14946ac179)
+- [Click Here](https://www.loom.com/share/ae186ef1e3af414d98192b9c474b3630)
 
```

Comment:
- [Optional] Dear @Amazinggracee, I see you have made adjustments to your video documentation, and it's great. But I believe it could be more organized in a way to make people understand everything there is to it. I suggest you follow the following structure while documenting a project. 
- 1. clear and objective description of the project in professional English
- 2. demo of the project, highlighting all the project features.
- 3. highlight a piece of the code the student finds interesting or is proud of making.
**This way, it becomes easier for your audience to follow along.**

Reply:
Thank you @TadesseDev, I will re-do the video.